.tito/lib/origin/builder/__init__.py

```diff
@@ -55,15 +55,37 @@ def _setup_test_specfile(self):
             )
             # Custom Openshift v3 stuff follows, everything above is the standard
             # builder
-            cmd = '. ./hack/common.sh ; echo $(os::build::ldflags)'
+            
+            ## Fixup os_git_vars
```

Comment:
Sorry, I meant if you write a Bash wrapper in `hack/common.sh` and call it the same way you're calling `os::build::ldflags`.